Mathlib/Analysis/Normed/Group/Continuity.lean

```diff
@@ -88,9 +98,28 @@ theorem continuous_norm' : Continuous fun a : E => ‖a‖ := by
 theorem continuous_nnnorm' : Continuous fun a : E => ‖a‖₊ :=
   continuous_norm'.subtype_mk _
 
-@[to_additive (attr := continuity, fun_prop) continuous_enorm]
-lemma continuous_enorm' : Continuous fun a : E ↦ ‖a‖ₑ :=
-  ENNReal.isOpenEmbedding_coe.continuous.comp continuous_nnnorm'
+end SeminormedGroup
+
+section Instances
```

Comment:
This section cannot move to `Basic`, and some proofs require `continuous_nnnorm'`, which is proven just a few lines above.